### PR TITLE
refactor: remove LowDB

### PR DESCRIPTION
**Pull request by jbrunton**

### CODE REVIEW

**Comment by jbrunton-ci-minion** [comment]
#### :tropical_drink: `up` on fs-db

   <details>
   <summary>Pulumi report</summary>

   ```
   Updating (fs-db)

View Live: https://app.pulumi.com/jbrunton/chat-demo/fs-db/updates/5


   pulumi:pulumi:Stack chat-demo-fs-db running 
   pulumi:pulumi:Stack chat-demo-fs-db running stack config: {
   pulumi:providers:aws aws  (0s) 
   pulumi:pulumi:Stack chat-demo-fs-db running read pulumi:pulumi:StackReference jbrunton/chat-demo-shared-infra/production
@ Updating....
   pulumi:pulumi:Stack chat-demo-fs-db running read pulumi:pulumi:StackReference jbrunton/chat-demo-shared-infra/production
   aws:ecs:Cluster chat-demo-fs-db  (0s) 
   aws:cloudwatch:LogGroup /ecs/chat-demo-fs-db  (0s) 
   aws:lb:TargetGroup chat-demo-fs-db  (0s) 
   aws:iam:Role chat-demo-fs-db  (0s) 
   aws:lb:ListenerRule chat-demo-fs-db  (0s) 
   aws:iam:RolePolicy chat-demo-fs-db-get-params  (0s) 
++ aws:ecs:TaskDefinition chat-demo-fs-db creating replacement (0s) [diff: ~containerDefinitions]
   aws:iam:RolePolicyAttachment chat-demo-fs-db  (0s) 
   aws:route53:Record chat-demo-fs-db  (0s) 
++ aws:ecs:TaskDefinition chat-demo-fs-db created replacement (0.89s) [diff: ~containerDefinitions]
+- aws:ecs:TaskDefinition chat-demo-fs-db replacing (0s) [diff: ~containerDefinitions]
+- aws:ecs:TaskDefinition chat-demo-fs-db replaced (0.00s) [diff: ~containerDefinitions]
~  aws:ecs:Service chat-demo-fs-db updating (0s) [diff: ~taskDefinition]
~  aws:ecs:Service chat-demo-fs-db updated (0.77s) [diff: ~taskDefinition]
-- aws:ecs:TaskDefinition chat-demo-fs-db deleting original (0s) [diff: ~containerDefinitions]
-- aws:ecs:TaskDefinition chat-demo-fs-db deleted original (0.29s) [diff: ~containerDefinitions]
   pulumi:pulumi:Stack chat-demo-fs-db  (17s) 1 message

Diagnostics:
 pulumi:pulumi:Stack (chat-demo-fs-db):
   stack config: {
    "stackName": "fs-db",
    "appName": "chat-demo-fs-db",
    "environment": "development",
    "tag": "HEAD-a6d9357",
    "protect": false,
    "domain": "fs-db.chat-demo.dev.jbrunton-aws.com",
    "publicUrl": "https://fs-db.chat-demo.dev.jbrunton-aws.com",
    "rootDomain": "jbrunton-aws.com"
   }

Outputs:
   publicUrl: "https://fs-db.chat-demo.dev.jbrunton-aws.com"

Resources:
   ~ 1 updated
   +-1 replaced
   2 changes. 10 unchanged

Duration: 23s

   ```
   
   </details>

File: server/src/data/low/auth-db.ts [hunk 1]
```diff
@@ -1,68 +1,79 @@
...
-    this.chain.get("verificationTokens");
+const initStore = (): AuthStore => ({
+  users: [],
+  sessions: [],
+  verificationTokens: [],
+  accounts: [],
+});
+
+export class AuthDB extends QuickDb<AuthSchema> {
+  //chain: ExpChain<QuickDbStore<AuthSchema>> = chain(this).get();
```
Comment (jbrunton) [inline server/src/data/low/auth-db.ts]: remove commented code

File: server/src/data/quickdb/adapters/file-adapter.ts [hunk 2]
```diff
@@ -0,0 +1,45 @@
+import * as fs from "fs";
+import * as path from "path";
+import { QuickDbAdapter, QuickDbStore } from "../types";
+
+const getFileName = (dbname: string) =>
+  path.join(process.cwd(), `${dbname}.json`);
```
Comment (jbrunton) [inline server/src/data/quickdb/adapters/file-adapter.ts]: for consistency, don't append `.json`

File: server/src/data/quickdb/quick-db.ts [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+import { QuickDbStore, QuickDbAdapter, QuickDbConfig } from "./types";
+
+export class QuickDb<T> {
+  private store: QuickDbStore<T>;
+  private readonly adapter: QuickDbAdapter<T>;
+
+  constructor({ adapter, initStore }: QuickDbConfig<T>) {
+    this.adapter = adapter;
+    // TODO: write store on init
+    this.store = cloneDeep(adapter.readStore() || initStore());
```
Comment (jbrunton) [inline server/src/data/quickdb/quick-db.ts]: write the store after init

**Pull request merged**